gobblin-core/src/main/java/gobblin/converter/csv/CsvToJsonConverter.java

```diff
@@ -53,11 +53,13 @@ public JsonArray convertSchema(String inputSchema, WorkUnitState workUnit) throw
   @Override
   public Iterable<JsonObject> convertRecord(JsonArray outputSchema, String inputRecord, WorkUnitState workUnit)
       throws DataConversionException {
-    InputStreamCSVReader reader =
-        new InputStreamCSVReader(inputRecord, workUnit.getProp(ConfigurationKeys.CONVERTER_CSV_TO_JSON_DELIMITER)
-            .trim().charAt(0), workUnit
-            .getProp(ConfigurationKeys.CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR,
-                ConfigurationKeys.DEFAULT_CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR).trim().charAt(0));
+    String enclosedStr =
+        workUnit.getProp(ConfigurationKeys.CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR,
+            ConfigurationKeys.DEFAULT_CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR);
```

Comment:
Why not just define `DEFAULT_CONVERTER_CSV_TO_JSON_ENCLOSEDCHAR` to `\0`? BTW: I saw `parser.ordinaryChar(enclosedChar);` in `InputStreamCSVReader`, which means it will treat the quoting character as ordinary character, is this expected?